src/main/java/eu/europeana/metis/sandbox/config/amqp/QueueConsumerConfig.java

```diff
@@ -100,9 +37,6 @@ protected SimpleRabbitListenerContainerFactory getSimpleRabbitListenerContainerF
       ConnectionFactory connectionFactory) {
     var factory = new SimpleRabbitListenerContainerFactory();
     configurer.configure(factory, connectionFactory);
```

Comment:
I have a suggestion now that I see the code is simplified. Adding here some extra parameters, to reduce code duplication in extended classes.
```
protected SimpleRabbitListenerContainerFactory getSimpleRabbitListenerContainerFactory(
      SimpleRabbitListenerContainerFactoryConfigurer configurer,
      ConnectionFactory connectionFactory, 
      Integer concurrentQueueConsumers,
      Integer maxConcurrentQueueConsumers,
      Integer messagePrefetchCount) {
    var factory = new SimpleRabbitListenerContainerFactory();
    configurer.configure(factory, connectionFactory);
    factory.setConcurrentConsumers(concurrentQueueConsumers);
    factory.setMaxConcurrentConsumers(maxConcurrentQueueConsumers);
    factory.setPrefetchCount(messagePrefetchCount);
    factory.setMessageConverter(messageConverter);
    return factory;
  }
```

Reply:
Sure I have applied it now.
But keep in mind that I was following the structure from before..